contracts/V1/StateChannelManagerEvents.sol

```diff
@@ -43,6 +43,8 @@ interface StateChannelManagerEvents {
 
     event ChannelStorageCleared(bytes32 indexed channelId, bytes32 latestJoinChannelBlockHash);
 
+    event ChannelClosed(bytes32 indexed channelId);
```

Comment:
This event is being emitted, but never collected on the client. I think we don't even need it, since people will leave when they observe they left or as you did when a new snapshot updates and they register 0 participants (they left) - so we can delete this explicit event emission unless we find a good reason to have it later